### DISCUSSION TITLE
Flickr importer

### DISCUSSION THREAD

**Discussion by aviks** [ente-io/ente#2611] [Enhancements]
I have many albums on Flickr (yes, I know!) so it would be great to have an importer that can import a flickr export dump in the app. Or, does anyone have any code/script lying around that might help?

**Comment by ua741** [comment]
You can request your photos and associated metadata from Flickr by following the instructions provided [here](https://gist.github.com/stungeye/e0897489fca28863c650fc33a95a11fb).

Ensure that you have corrected the metadata before importing them to Ente. If the EXIF data is missing from your export file, the files might appear in the incorrect order on Ente.

**Reply by aviks** [reply]
Yeah, I have the flickr export dump. I have the photos assigned to albums in flickr, while the export is flat. The album information is in the metadata json. I want to have the albums preserved when loading in ente. 

Also, that link seems to suggest the need for manual fixups for the metadata, which, with a few thousand photos, I'm not sure I'll be able to do.
